.gitmodules

```diff
@@ -2,4 +2,4 @@
 	path = aerospike-client-c
 	# url = git@github.com:aerospike/aerospike-client-c.git
 	url = https://github.com/aerospike/aerospike-client-c.git
-	branch = stage
+	branch = CLIENT-2294
```

Comment:
Double checking that this is set correctly.

Reply:
Yeah this is the right branch containing the C client's MRT work